server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -1062,26 +1062,25 @@ public UserDto editUserProfile(UserDto userDto) {
         return response.getBody();
     }
 
-    public CredentialsDto provisionCredentials (String applicationId, String credentials) {
+    public CredentialsDto provideCredentials (String applicationId, String credentialsBody) {
         MultiValueMap<String, Object> parameters = new LinkedMultiValueMap<>();
         parameters.add("applicationId", applicationId);
-        parameters.add("credentials", credentials);
-        return this.restTemplate.postForObject(restTemplate.getUrl() + "provisionCredentials", parameters, CredentialsDto.class);
+        parameters.add("credentialsBody", credentialsBody);
+        return this.restTemplate.postForObject(restTemplate.getUrl() + "provideCredentials", parameters, CredentialsDto.class);
     }
 
-    public void provisionRegistrationInfo(String applicationId, String credentialsId, Integer serverSideEPProfileVersion, String serverSideEPProfileBody){
+    public void provideRegistration(String applicationId, String credentialsId, Integer serverProfileVersion, String serverProfileBody){
         MultiValueMap<String, Object> parameters = new LinkedMultiValueMap<>();
         parameters.add("applicationId", applicationId);
         parameters.add("credentialsId", credentialsId);
-        parameters.add("serverSideEPProfileVersion", serverSideEPProfileVersion);
-        parameters.add("serverSideEPProfileBody", serverSideEPProfileBody);
-
-        this.restTemplate.postForLocation(restTemplate.getUrl() + "provisionRegistrationInfo", parameters);
-
+        parameters.add("serverProfileVersion", serverProfileVersion);
+        parameters.add("serverProfileBody", serverProfileBody);
```

Comment:
I don't like the "server profile" shortcut for "server-side EP profile", because it makes people think that the profile somehow actually relates to Kaa server, not the EP. If you feel like "server-side EP profile" is too cumbersome, we can think of a different shortcut. "Managed profile" could be good.